module/zfs/zvol.c

```diff
@@ -108,6 +109,27 @@ static struct hlist_head *zvol_htable;
 
 static struct ida zvol_ida;
 
+typedef struct zvol_aggsum_stats {
+	aggsum_t zvol_as_writes;
+	aggsum_t zvol_as_nwritten;
+	aggsum_t zvol_as_reads;
+	aggsum_t zvol_as_nread;
+} zvol_aggsum_stats_t;
+
+typedef struct zvol_kstats {
+	kstat_named_t zk_writes;
+	kstat_named_t zk_nwritten;
+	kstat_named_t zk_reads;
+	kstat_named_t zk_nread;
+} zvol_kstats_t;
+
+static zvol_kstats_t empty_zvol_kstats = {
```

Comment:
in theory, we could add the zvol name here

Reply:
I'm not sure how useful would that be here.
If you were able to get the objsetID for a dataset, I assume you already had its name.
If there is a practical use-case that I'm missing though I'd be open to including this change.